src/frame/types.ts

```diff
@@ -77,7 +77,7 @@ export function convertToFrame(json: any) {
  */
 export type FrameButtonMetadata =
   | {
-      action: 'link' | 'mint';
+      action: 'link' | 'mint' | 'tx';
```

Comment:
tiny nitpick: can we have `txn` instead of `tx` (Texas?)

Reply:
That's a great point!

OnchainKit will follow the Frame spec https://warpcast.notion.site/Frame-Transactions-Public-Draft-v2-9d9f9f4f527249519a41bd8d16165f73 

if they change it to `txn`, we will do too.

Feel free to ask there if they will consider `txn` vs `tx.